lib/screens/repositories_screen.dart

```diff
@@ -22,12 +22,13 @@ class _RepositoriesScreenState extends State<RepositoriesScreen> {
     "TypeScript",
   ];
   String? _selectedLanguage;
-  List<String> repositories = [];
+  List<String> _repositories = [];
 
-  void _didSelectItem(String item) {
+  void _didSelect(String item) {
```

Comment:
It seems you forgot to put {} around String item. When you put those around it, it becomes a named argument instead of a positional argument.

Reply:
done